src/screens/DataApi/DataApi.tsx

```diff
@@ -36,6 +39,25 @@ export const sidebarSections: TocItem[] = [
   },
 ];
 
+const DataCoverageSection: React.FC<{}> = () => {
+  return (
+    <Grid container spacing={1}>
+      <Grid item xs={12}>
+        <DataCoverageTable rows={FIELDS_PRETTY_NAMES} />
+      </Grid>
+      <Grid item xs={12}>
```

Comment:
nit: We can probably move the link inside the other `<Grid item xs={12}>` container (unless you just want the spacing)